modules/installation/pages/general-requirements.adoc

```diff
@@ -47,6 +47,10 @@ Depending on your organization's setup, you might also need to activate your sub
 
 From {slsa}{nbsp}15{nbsp}SP1, {susemgr} Server and Proxy is available as a base product, and can be installed with the {slsa} Unified Installer.
```

Comment:
Unrelated, but should that use `{slsa} {nbsp}{sles-version}{nbsp} {sp-version}`?

Reply:
I wrote it that way because we don't want that version to change. Even after we're based on a newer version, it will still be true that it's a base product from 15 SP1.

Reply:
That's right, but my understanding is that those values will always come from the yaml files.

@jcayouette ?